<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditWindow</class>
 <widget class="QDialog" name="CreditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_total_credit_amount">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <double>1000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_interest_rate">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>99.989999999999995</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_monthly_payment">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: black;
   background-color: white;
   border: 2px solid orange;
	border-radius: 10px;
}
</string>
   </property>
   <property name="text">
    <string>monthly payment: 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_overpayment_on_credit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: black;
   background-color: white;
   border: 2px solid orange;
	border-radius: 10px;
}
</string>
   </property>
   <property name="text">
    <string>overpayment on credit: 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_payment">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: black;
   background-color: white;
   border: 2px solid orange;
	border-radius: 10px;
}
</string>
   </property>
   <property name="text">
    <string>total payment: 0</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_year">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calc">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>300</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: black;
   background-color: white;
   border: 2px solid blue;
	border-radius: 10px;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #9799fc);
}</string>
   </property>
   <property name="text">
    <string>calculate</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_month">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1080</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>annuity</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>differentiated</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: black;
   background-color: white;
   border: 2px solid blue;
	border-radius: 10px;
}
</string>
   </property>
   <property name="text">
    <string>total credit amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: black;
   background-color: white;
   border: 2px solid blue;
	border-radius: 10px;
}
</string>
   </property>
   <property name="text">
    <string>interest rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: black;
   background-color: white;
   border: 2px solid blue;
	border-radius: 10px;
}
</string>
   </property>
   <property name="text">
    <string>year</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: black;
   background-color: white;
   border: 2px solid blue;
	border-radius: 10px;
}
</string>
   </property>
   <property name="text">
    <string>month</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: black;
   background-color: white;
   border: 2px solid blue;
	border-radius: 10px;
}
</string>
   </property>
   <property name="text">
    <string>type</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_AC">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>230</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: black;
   background-color: white;
   border: 2px solid blue;
	border-radius: 10px;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #9799fc);
}</string>
   </property>
   <property name="text">
    <string>AC</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
